content/chainguard/chainguard-images/pricing.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Chainguard Container Catalog Pricing"
+linktitle: "Catalog Pricing"
+type: "article"
+description: "Explains Chainguard Container catalog pricing, repository quotas, renaming, and sync, and helps you understand how to make the most of your subscription."
+date: 2025-08-19T08:49:31+00:00
+lastmod: 2025-08-19T08:49:31+00:00
+draft: false
+tags: ["Chainguard Containers", "pricing"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 055
+toc: true
+---
+
+Catalog pricing is Chainguard's pricing model for container usage that replaces the manual ticket process for adding images to an organization. This enables you to self-service request individual images in the console to be added to your repository from the wider Chainguard catalog.
```

Comment:
Since this is documentation and not an announcement, I don't think we need to say it replaces another process.

Reply:
I thought that was part of why end users would care. Streamlining the process by removing a typical step (that still exists in our classic pricing model) is part of the value proposition.

If you still think it needs to go, I'll remove it.